_includes/graphql/getting-started.md

```diff
@@ -0,0 +1,23 @@
+# Getting Started
+
+[GraphQL](https://graphql.org/), developed by Facebook, is an open-source data query and manipulation language for APIs. In addition to the traditional [REST API](/rest/guide/), Parse Server automatically generates a GraphQL API based on your current application schema.
+
+The easiest way to run the Parse GraphQL Server is using the CLI:
+
+```bash
+$ npm install -g parse-server mongodb-runner
+$ mongodb-runner start
+$ parse-server --appId APPLICATION_ID --masterKey MASTER_KEY --databaseURI mongodb://localhost/test --mountGraphQL --mountPlayground
+```
```

Comment:
I don't mind if this isn't done in this PR, but do we need to provide different instructions for Windows users?

Reply:
Sure. Do you have an Windows machine to test by any chance? :)

Reply:
no. @oallouch I believe you use a Windows machine? - if you get a chance it would be great if you could test this out.